core/backend-tests/README.md

```diff
@@ -0,0 +1,11 @@
+# @itwin/core-backend
```

Comment:
Still contains core-backend readme file

Reply:
Could you add a note in `core-backend` to write tests in this package?

Reply:
Fixed; let me know if you think more is warranted.